eureka-client/src/main/java/com/netflix/discovery/converters/wrappers/DecoderWrapper.java

```diff
@@ -0,0 +1,14 @@
+package com.netflix.discovery.converters.wrappers;
+
+import java.io.IOException;
+import java.io.InputStream;
+
+/**
+ * @author David Liu
+ */
+public interface DecoderWrapper extends CodecWrapperBase {
```

Comment:
Shall we drop 'wrapper' term, as this is more like Eureka codec API now? (for example EurekaCodecBase, ...)